migrations/1575287807413-remove_create_permissions_thuringia.js

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line no-unused-vars
```

Comment:
For future changes, please try to not change the migration files (this change here ist okay). They represent changes to the database that were run once in the past and they stay in the code for reference only.

Reply:
I see, makes sense. 👍